compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -221,6 +221,18 @@ impl<'a: 'ast, 'ast, 'tcx> LateResolutionVisitor<'a, '_, 'ast, 'tcx> {
             let mut span_label = None;
             let item_ident = path.last().unwrap().ident;
             let item_span = item_ident.span;
+            if let PathSource::Delegation = source {
```

Comment:
Not sure why a special error is required here.
What happens if it is removed?

Reply:
In fact, this is not necessary. The message will be "cannot find function 'function' in 'module'".